_partials/_livesync-configure-source-database.md

```diff
@@ -16,26 +18,54 @@
    However, you can also have:
 
    - **A viable unique index**: each table has a unique, non-partial, non-deferrable index that includes only columns
-      marked as `NOT NULL`. If a `UNIQUE` index does not exists, create one to assist the migration. You can delete it after
-      live sync. For each table, set `REPLICA IDENTITY` to the viable unique index:
+     marked as `NOT NULL`. If a `UNIQUE` index does not exists, create one to assist the migration. You can delete it after
+     live sync. For each table, set `REPLICA IDENTITY` to the viable unique index:
 
       ```sql
-       psql -X -d $SOURCE -c 'ALTER TABLE <table name> REPLICA IDENTITY USING INDEX <_index_name>'
+       psql $SOURCE -c 'ALTER TABLE <table name> REPLICA IDENTITY USING INDEX <_index_name>'
       ```
 
    - **No primary key or viable unique index**: use brute force. For each table, set `REPLICA IDENTITY` to `FULL`:
-   
+
        ```sql
-       psql -X -d $SOURCE -c 'ALTER TABLE <table name> REPLICA IDENTITY FULL'
+       psql $SOURCE -c 'ALTER TABLE <table name> REPLICA IDENTITY FULL'
        ```
-      For each `UPDATE` or `DELETE` statement, PostgreSQL reads the whole table to find all matching rows.
-      This results in significantly slower replication. If you are expecting a large number of `UPDATE` or `DELETE`
-      operations on the table, best practice is to not use `FULL`
-   
-      To capture only `INSERT` and ignore `UPDATE`s and `DELETE`s, use a
-      [publish config](https://www.postgresql.org/docs/current/sql-createpublication.html#SQL-CREATEPUBLICATION-PARAMS-WITH-PUBLISH)
-      while [creating the publication][lives-sync-specify-tables].
+     For each `UPDATE` or `DELETE` statement, PostgreSQL reads the whole table to find all matching rows.
+     This results in significantly slower replication. If you are expecting a large number of `UPDATE` or `DELETE`
+     operations on the table, best practice is to not use `FULL`
+
+     To capture only `INSERT` and ignore `UPDATE`s and `DELETE`s, use a
+     [publish config](https://www.postgresql.org/docs/current/sql-createpublication.html#SQL-CREATEPUBLICATION-PARAMS-WITH-PUBLISH)
+     while [creating the publication][lives-sync-specify-tables].
+
+
+1. **Create a user for livesync and assign permissions**
+
+   1. Create `<livesync username>`:
```

Comment:
@billy-the-fish 
Customer can reuse the existing user instead of creating new one just for the sake of livesync.

The only requirement is to have `rds_replication and  `CREATE ON DATABASE <database name> to <livesync username>;` grants.